app/views/home/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if user_signed_in? %> 
+ <div> Welcome <%= current_user.email %> </div> 
+ <%= link_to "Categories", categories_path %>
+  <%= button_to "Sign out", destroy_user_session_path, method: :delete %> 
+<% else %>
+<div class="min-vh-100">
+<div class="d-flex container justify-content-end"><%= link_to "Sign in", new_user_session_path, class: "btn btn-primary btn-" %></div>
+
+  <div class="position-absolute start-50 top-50 translate-middle">
+    <h1>PaySafe<h1>
+  </div>
+   <div class="position-absolute start-50 bottom-0 translate-middle"><%= link_to "Register", new_user_registration_path, class: "btn btn-secondary btn-block" %></div>
+ 
+</div>
+  
+  
+ 
+<% end %>
+ 
```

Comment:
- [ ] Consider adding a splash page when the user opens your app like the following image:
>![image](https://user-images.githubusercontent.com/53402600/205068434-4e1b4276-298a-4dc4-ba67-1223bf8a727b.png)